demisto_sdk/commands/find_dependencies/tests/find_dependencies_test.py

```diff
@@ -237,7 +281,131 @@ def test_collect_scripts_depends_on_two_integrations(self, id_set):
+                        "ad-search": "activedir"
+                    },
+                    "pack": "Active_Directory_Query"
+                }
+            }
+        ]
+
+        found_result = PackDependencies._collect_scripts_dependencies(pack_scripts=test_input,
+                                                                      id_set=id_set,
+                                                                      verbose_file=VerboseFile(),
+                                                                      )
+
+        assert IsEqualFunctions.is_sets_equal(found_result, expected_result)
+
+    def test_collect_scripts_script_executions(self, id_set):
+        """
+        Given
+            - A script entry in the id_set containing a script_executions, e.g: demisto.executeCommand(<command>).
+
+        When
+            - Building dependency graph for pack.
+
+        Then
+            - Extracting the pack that the script depends on.
+            - Should recognize the pack.
+        """
+        expected_result = {('Active_Directory_Query', True)}
+
+        test_input = [
+            {
+                "DummyScript": {
+                    "name": "ADIsUserMember",
+                    "file_path": "Packs/DeprecatedContent/Scripts/script-ADIsUserMember.yml",
+                    "deprecated": False,
+                    "depends_on": [
+                    ],
+                    "script_executions": [
+                        "ADGetUser",
+                    ],
+                    "pack": "Active_Directory_Query"
+                }
+            }
+        ]
+
+        found_result = PackDependencies._collect_scripts_dependencies(pack_scripts=test_input,
+                                                                      id_set=id_set,
+                                                                      verbose_file=VerboseFile(),
+                                                                      )
+
+        assert IsEqualFunctions.is_sets_equal(found_result, expected_result)
+
+    def test_collect_scripts_command_to_integrations_and_script_executions(self, id_set):
+        """
+        Given
+            - A script entry in the id_set containing command_to_integrations and script_executions.
+
+        When
+            - Building dependency graph for pack.
+
+        Then
+            - Extracting the packs that the script depends on.
+            - Should recognize both packs.
+        """
+        expected_result = {('Active_Directory_Query', True), ('Cherwell', True)}
+
+        test_input = [
+            {
+                "DummyScript": {
+                    "name": "double_dependency",
+                    "file_path": "Packs/DeprecatedContent/Scripts/script-ADIsUserMember.yml",
+                    "deprecated": False,
+                    "depends_on": [
+                    ],
+                    "command_to_integration": {
+                        "cherwell-get-business-object": "Cherwell"
```

Comment:
as you already have a test for `command_to_integration`, use a command that appears in multiple integrations.
then, those packs should not be mandatory